[ 2024-04-04 04:23:03 ] ERROR in tech.iresponse.l.b : tech.iresponse.d.b: No suppression link found !
[ 2024-04-04 04:23:49 ] ERROR in tech.iresponse.break.do : java.io.FileNotFoundException: /usr/irresponse/public/media/39da2cda0b8e6f6.png (No such file or directory)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:162)
	at tech.iresponse.long.if.do(Unknown Source)
	at tech.iresponse.long.if.do(Unknown Source)
	at tech.iresponse.do.if.int.do(Unknown Source)
	at tech.iresponse.do.if.int.run(Unknown Source)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

[ 2024-04-04 04:23:49 ] ERROR in tech.iresponse.break.do : java.io.FileNotFoundException: /usr/irresponse/public/media/211cc756d351388.png (No such file or directory)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:162)
	at tech.iresponse.long.if.do(Unknown Source)
	at tech.iresponse.long.if.do(Unknown Source)
	at tech.iresponse.do.if.int.do(Unknown Source)
	at tech.iresponse.do.if.int.run(Unknown Source)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

[ 2024-04-04 04:27:23 ] ERROR in tech.iresponse.break.do : org.postgresql.util.PSQLException: ERROR: duplicate key value violates unique constraint "pg_type_typname_nsp_index"
  Detail: Key (typname, typnamespace)=(charter, 32777) already exists.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2440)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2183)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:308)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:143)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:120)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:384)
	at tech.iresponse.const.for.if(Unknown Source)
	at tech.iresponse.const.do.package(Unknown Source)
	at tech.iresponse.byte.new.do(Unknown Source)
	at tech.iresponse.byte.new.do(Unknown Source)
	at tech.iresponse.this.do.for(Unknown Source)
	at tech.iresponse.Bootstrap.main(Unknown Source)

[ 2024-04-04 04:27:24 ] ERROR in tech.iresponse.break.do : org.postgresql.util.PSQLException: ERROR: duplicate key value violates unique constraint "c_pk_id_charter"
  Detail: Key (id)=(1) already exists.
  Where: COPY charter, line 2
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2440)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2183)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:308)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:143)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:120)
	at com.mchange.v2.c3p0.impl.NewProxyPreparedStatement.executeUpdate(NewProxyPreparedStatement.java:384)
	at tech.iresponse.const.for.if(Unknown Source)
	at tech.iresponse.byte.new.do(Unknown Source)
	at tech.iresponse.byte.new.do(Unknown Source)
	at tech.iresponse.this.do.for(Unknown Source)
	at tech.iresponse.Bootstrap.main(Unknown Source)

[ 2024-04-04 04:29:17 ] ERROR in tech.iresponse.l.b : tech.iresponse.d.b: No suppression link found !
[ 2024-04-04 04:33:11 ] ERROR in tech.iresponse.l.b : tech.iresponse.d.b: No suppression link found !
